pom.xml

```diff
@@ -673,6 +677,9 @@
                                     <goals>
                                         <goal>prepare-agent</goal>
                                     </goals>
+                                    <configuration>
+                                        <propertyName>jacoco.arg.line</propertyName>
```

Comment:
❓ How does this property work? Does it get set automagically by the plugin? Do we need to set its value somewhere?

Reply:
This is filled automatically by the plugin.